Zend/tests/type_declarations/mixed/syntax/mixed_parameter_error.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Test that the mixed parameter type is not valid with union types
+--FILE--
+<?php
+
+function foo(mixed|int|null $a)
+{
+}
+
+?>
+--EXPECTF--
+Fatal error: Duplicate type int is redundant in %s on line %d
```

Comment:
What about
```suggestion
Fatal error: Type mixed used with other types is redundant in %s on line %d
```
?

Because this error message would make me think that I have something like this in my code:
```php
function foo(int|int|null $a) {
```

Reply:
Yes, I'm totally fine with this change! I just didn't have the time yet to fully work out the details and the tests 😊

Reply:
Done :)